src/main/init.f90

```diff
@@ -75,8 +75,38 @@ subroutine init_model(options,domain,boundary)
         if (this_image()==1) write(*,*) "Initializing boundary condition data structure"
         call boundary%init(options)
 
-        if (this_image()==1) write(*,*) "Reading Initial conditions from boundary dataset"
-        call domain%get_initial_conditions(boundary, options)
+        if(options%parameters%external_files/="MISSING") then
```

Comment:
YEs, agreed, but now it was preventing the correct restart. 
Btw I am having issues with restarting runs, where the first timestamp after the restart is wrong, it has the timestamp of the start of the run. The file is named correctly though, and the 2nd file after the restart all is well again. Strange because I remember restarts used to work fine. To be continued.